### PR TITLE
Update code to handle the IP range expansion

### PR DESCRIPTION
**Pull request by ShubhamRasal**
- Update the `main.go` to handle the case to IP range expansion
- Rename the variable `CIDRFromIPRange` to `IPRangeInput`
- Add the if block which handles the case when Input is IP range.
- Add the IP expansion example in README.md

### CODE REVIEW

File: cmd/mapcidr/main_test.go [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+				IPRangeInput: true,
+			},
+			expectedOutput: []string{"10.40.0.0/25", "10.40.0.128/25"},
+		},
+	}
+	var wg sync.WaitGroup
+
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			wg.Add(2)
```
Comment (jdk2588) [inline cmd/mapcidr/main_test.go]: A small change, just add like wg.Add(1) before running the go function and then do another wg.Add(1) before calling the closure?

File: cmd/mapcidr/main_test.go [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+
+			go process(&wg, tt.chancidr, tt.outputchan)
+
+			var outputlist []string
+			// get output list
+			go func() {
+				for output := range tt.outputchan {
+					outputlist = append(outputlist, output)
+				}
+				wg.Done()
```
Comment (jdk2588) [inline cmd/mapcidr/main_test.go]: call like `defer wg.Done()`?

**Review by jdk2588 — CHANGES_REQUESTED** [review]

File: cmd/mapcidr/main_test.go (hunk 2 above)
Comment (ShubhamRasal) [reply cmd/mapcidr/main_test.go]: If I put this as defer then I have to move if DeepEqual block out of `t.Run`

Comment (ShubhamRasal) [reply cmd/mapcidr/main_test.go]: It has to be that way only, coz we need to wait for the output list before checking it

Comment (jdk2588) [reply cmd/mapcidr/main_test.go]: Ummm

Why can't it be like 
```
go func() {
                defer wg.Done()
	        for output := range tt.outputchan {
			outputlist = append(outputlist, output)
		}
}()
```

Comment (ShubhamRasal) [reply cmd/mapcidr/main_test.go]: whoo.. my bad... I was looking at wg.wait

Comment (ShubhamRasal) [reply cmd/mapcidr/main_test.go]: done...

**Review by jdk2588 — APPROVED** [review]

**Pull request merged**